routing/following_info.hpp

```diff
@@ -84,9 +84,8 @@ class FollowingInfo
   turns::PedestrianDirection m_pedestrianTurn;
   //@}
 
-  // Current speed limit.
-  // If no info about speed limit then m_speedLimit == "" and m_speedLimitUnitsSuffix == "".
-  std::string m_speedLimit;
-  std::string m_speedLimitUnitsSuffix;
+  // Current speed limit in meters per second.
+  // If no info about speed limit then m_speedLimitMps < 0.
+  double m_speedLimitMps = -1.0;
```

Comment:
Лучше всё-таки везде длиннее, но однозначнее, тогда комментарий не нужен.
```suggestion
  double m_speedLimitMetersPerSecond = -1.0;
```

Reply:
Мой имхо MetersPerSecond  - перебор. С чем это можно перепутать?

Reply:
Как с чем? Я читаю всегда сначала как miles per second, стандартная аббревиатура MPS.

Reply:
И многие американцы/англичане тоже могут так читать.

Reply:
Саша, miles per second? Та на ракете летаешь? Нет такой стандартной аббревиатуры :)

Reply:
Мили - это всегда Mi,
MPH - единственное исключение.

Reply:
Столько спора из-за _более понятной_ строчки ) Или мой вариант _менее понятен_?

Reply:
Тогда везде надо исправлять. Необходимо единообразие как раз во избежание недопонимания.
У нас скорость как правило именно в метрах в секунду и почти везде постфикс mps.

Reply:
Лично у меня такие длинные название переменных m_speedLimitMetersPerSecond вызывают диссонанс.